joingroup_test.go

```diff
@@ -130,12 +131,20 @@ func TestSaramaCompatibility(t *testing.T) {
 		//
 		// See consumer_group_members_test.go
 		//
+		groupMemberMetadataBad = []byte{
```

Comment:
Is "bad" here just mean it's missing ownedpartitions and is the backwards compatible version? I wonder if there's a different name we can use here and in the test to make this more clear